### PR TITLE
RNA STAR diploid mode

### PR DESCRIPTION
**Pull request by TomHarrop**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

---

Adds STAR diploid mode from 2.7.11a.

Closes #5653

### CODE REVIEW

**Comment by TomHarrop** [comment]
@bgruening if this looks OK to you I'll generate some test data and add tests

**Comment by TomHarrop** [comment]
Passes linting locally with planemo version 0.75.19

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks @TomHarrop

File: tools/rgrnastar/macros.xml [hunk 1]
```diff
@@ -15,14 +15,14 @@
     or by looking for the versionGenome parameter in source/parametersDefault
     of STAR's source code -->
     <token name="@IDX_VERSION@">2.7.4a</token>
-    <token name="@IDX_VERSION_SUFFIX@">1</token>
+    <token name="@IDX_VERSION_SUFFIX@">0</token>
```
Comment (lldelisle) [inline tools/rgrnastar/macros.xml:18]: Oops... This should not have been changed! @bgruening @TomHarrop Should we fix that immediately?

Comment (bgruening) [reply tools/rgrnastar/macros.xml:18]: Yes, lets fix it now. Thanks for the catch.

Comment (TomHarrop) [reply tools/rgrnastar/macros.xml:18]: Don't know why I did this 

https://github.com/galaxyproject/tools-iuc/pull/5778

**Pull request merged**